### PR TITLE
feat: new metric to add exporter version

### PR DESCRIPTION
**Pull request by dmachard**
This PR add a new metric to follow up the version of this exporter.
No breaking changes.

### CODE REVIEW

**Review by mehdy — DISMISSED** [review]
LGTM.
@clwluvw anything to add?

**Review by clwluvw — CHANGES_REQUESTED** [review]
We discussed that internally, it'd be better to switch this version handling in another package so we don't need to pass these as an arg, and would also be cleaner. Normally we can use https://github.com/prometheus/common/tree/main/version for that which has also a function to register it as a metric.

Would it be possible for you to do this change?
You can also pick this https://github.com/prometheus/node_exporter/blob/master/node_exporter.go#L122 as an example.

**Comment by dmachard** [comment]
Thank for the tip, you are right it's more cleaner.
Below the new metrics with prometheus version package.

```
# HELP keepalived_exporter_build_info A metric with a constant '1' value labeled by version, revision, branch, and goversion from which keepalived_exporter was built.
keepalived_exporter_build_info{branch="",goversion="go1.18.1",revision="",version="689046e"} 1
```

File: cmd/keepalived-exporter/main.go [hunk 1]
```diff
@@ -50,6 +51,9 @@ func main() {
 	keepalivedCollector := collector.NewKeepalivedCollector(*keepalivedJSON, *keepalivedCheckScript, c)
 	prometheus.MustRegister(keepalivedCollector)
 
+	promver.Version = version
```
Comment (clwluvw) [inline cmd/keepalived-exporter/main.go:54]: Thanks! Just two things are remaining to get it done.
1. Please remove the vars (`version`, `commit`, `buildTime`) from the main package and edit the Makefile to set directly to the `prometheus/common/version` package vars.
2. Please edit the Makefile to set `branch` and `revision` as well (Let's not expose a metric with empty labels).

**Review by clwluvw — CHANGES_REQUESTED** [review]

**Comment by dmachard** [comment]
done,   updated output version:

```
$ ./keepalived-exporter -version
INFO[0000] Keepalived Exporter                           build_time="2023-04-19_08:43:43_+0200" commit=9c200ce version=9c200ce
```

```
# HELP keepalived_exporter_build_info A metric with a constant '1' value labeled by version, revision, branch, and goversion from which keepalived_exporter was built.
keepalived_exporter_build_info{branch="feat-add-version",goversion="go1.18.1",revision="9c200ce",version="9c200ce"} 1
```

**Review by clwluvw — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**